tests/test_watson.py

```diff
@@ -716,17 +785,19 @@ def json(self):
 
     assert len(watson.frames) == 2
 
-    assert watson.frames[0].id == '1'
-    assert watson.frames[0].project == 'foo'
-    assert watson.frames[0].start.timestamp == 3
-    assert watson.frames[0].stop.timestamp == 4
-    assert watson.frames[0].tags == ['A']
-
-    assert watson.frames[1].id == '2'
-    assert watson.frames[1].project == 'bar'
-    assert watson.frames[1].start.timestamp == 4
-    assert watson.frames[1].stop.timestamp == 5
-    assert watson.frames[1].tags == []
+    frame = watson.frames.get_by_index(0)
+    assert frame.id == '1'
+    assert frame.project == 'foo'
+    assert frame.start.timestamp == 3
+    assert frame.stop.timestamp == 4
+    assert frame.tags == ['A']
+
+    frame = watson.frames.get_by_index(1)
+    assert frame.id == '2'
+    assert frame.project == 'bar'
+    assert frame.start.timestamp == 4
+    assert frame.stop.timestamp == 5
+    assert frame.tags == []
 
 
```

Comment:
Could we add a test to see that loading frames stored with the old format works ?